Decisions/scs-XXXX-v1-requirements-for-container-registry.md

```diff
@@ -0,0 +1,382 @@
+- Maturity
+- Security
+- Activity
+- Lock-in risk assessment
+
+Each selected OSS project is evaluated based on the above checks, and it is classified
+into one of three categories as follows:
+
+- :heavy_check_mark: The project passed all OSS health checks and will be considered 
+  further as a valid candidate.
+
+- :grey_question: The project passed almost all OSS health checks. 
+  There is place for improvement, but the missing points are not crucial from the OSS
+  health check perspective. The project will be considered further as a valid candidate.
+
+- :x: The project does not pass the OSS health checks. Some OSS health check 
+  showstoppers have been found (e.g. open core software, not actively maintained). 
+  The project is filtered at this stage and won't be considered further.
+
+Refer to the list of evaluated projects with their classified categories and comments.
+
+- :heavy_check_mark: [Harbor][harbor]
+  - Harbor project meets all "four opens"
+    - Source code is open and available under the [Apache 2 license](https://github.com/goharbor/harbor/blob/main/LICENSE)
+    - Community is [open](https://github.com/goharbor/harbor#community), structured and 
+      well organized via [workgroups](https://github.com/goharbor/community) and 
+      various communications channels e.g. Slack, mailing lists, etc. 
+      (#harbor Slack channel contains 3k+ members)
+    - The development process is open via GitHub issues and well described in the 
+      [contributing](https://github.com/goharbor/harbor/blob/main/CONTRIBUTING.md) 
+      document
+    - The design process is open via GitHub issues. Proposals are [public](https://github.com/goharbor/community/tree/main/proposals).
+      The decision process is well described as well. The project's roadmap is 
+      available in the [roadmap](https://github.com/goharbor/harbor/blob/main/ROADMAP.md) document
+  - Maturity is on the CNCF [graduation](https://www.cncf.io/projects/harbor/) level.
+    CNCF graduated projects are considered to be stable, widely adopted and production-ready
+  - Security
+    - The security disclosure and response policy is well described in the project's
+      [security](https://github.com/goharbor/harbor/blob/main/SECURITY.md) document
+    - The code is reviewed within a standard PR process
+  - Activity
+    - 250+ contributors, 4k+ forks, 13k+ GitHub stars
+    - The project has been [adopted](https://github.com/goharbor/harbor/blob/main/ADOPTERS.md) 
+      by many companies that run Harbor in their production environments
+    - The project collaborates with other communities and projects
+      (see [Partners of Harbor](https://goharbor.io/community/) section of the 
+      project's website)
+    - The project is visible and actively contributes to various conferences, e.g.
+      [FOSDEM 22](https://goharbor.io/blog/harbor-at-fosdem-2022/),
+      [KubeCon Europe](https://www.youtube.com/watch?v=MGNtPHQYP14), etc. 
```

Comment:
This seems to be a talk about dragonfly (same as below)?

Reply:
fixed 1100a04fa12721b3fb8a499b4b39a46f69904f4c